Training/python/DataLoader_test.py

```diff
@@ -0,0 +1,120 @@
+# The environment on Centos 7 is:
+# source /cvmfs/sft.cern.ch/lcg/views/SetupViews.sh LCG_99 x86_64-centos7-gcc10-opt
+import ROOT as R
+import numpy as np
+import time
+import sys
+import config_parse
+
+R.gROOT.ProcessLine(".include ../../..")
+
+print("Compiling DatLoader enume and setup classes...")
+R.gInterpreter.ProcessLine(config_parse.create_settings("../configs/training_v1.yaml",verbose=False))
+
+print("Compiling scaling params classes...")
+R.gInterpreter.ProcessLine(config_parse.create_scaling_input("../configs/scaling_test.json",verbose=False))
+
+print("Compiling DataLoader_main...")
+R.gInterpreter.ProcessLine('#include "../interface/DataLoader_main.h"')
+
+n_tau          = R.Setup.n_tau
+n_inner_cells  = R.Setup.n_inner_cells
+n_outer_cells  = R.Setup.n_outer_cells
+n_fe_tau    = R.Setup.n_TauFlat
+n_pf_el     = R.Setup.n_PfCand_electron
+n_pf_mu     = R.Setup.n_PfCand_muon
+n_pf_chHad  = R.Setup.n_PfCand_chHad
+n_pf_nHad   = R.Setup.n_PfCand_nHad
+n_pf_gamma  = R.Setup.n_pfCand_gamma
+n_ele       = R.Setup.n_Electron
+n_muon      = R.Setup.n_Muon
+tau_types   = R.Setup.tau_types_names.size()
+
+data_loader = R.DataLoader()
+print("Number of all entries / batch size: ", data_loader.GetEntries()/n_tau)
+
+n_batches = 500
+n_batches_store = 5
+
+# To convert list to tensor
+def to_tensor(data):
+
+    y = np.zeros((n_tau,tau_types))
+    w = np.zeros((n_tau))
+    x_tau = np.zeros((n_tau,n_fe_tau))
+
+    x_outer_1 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_pf_el))
+    x_outer_2 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_pf_mu))
+    x_outer_3 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_pf_chHad))
+    x_outer_4 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_pf_nHad))
+    x_outer_5 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_pf_gamma))
+    x_outer_6 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_ele))
+    x_outer_7 = np.zeros((n_tau, n_outer_cells, n_outer_cells, n_muon))
+
+    x_inner_1 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_pf_el))
+    x_inner_2 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_pf_mu))
+    x_inner_3 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_pf_chHad))
+    x_inner_4 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_pf_nHad))
+    x_inner_5 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_pf_gamma))
+    x_inner_6 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_ele))
+    x_inner_7 = np.zeros((n_tau, n_inner_cells, n_inner_cells, n_muon))
+
+    y = np.frombuffer(data.y_onehot.data(), dtype=np.float32, count=data.y_onehot.size()).reshape((n_tau, tau_types))
+    w = np.frombuffer(data.weight.data(), dtype=np.float32, count=data.weight.size())
+    x_tau = np.frombuffer(data.x_tau.data(), dtype=np.float32, count=data.x_tau.size()).reshape((n_tau,n_fe_tau))
+
+    x_outer_1 = np.frombuffer(data.x_grid[0][0].data(), dtype=np.float32, count=data.x_grid[0][0].size()).reshape((n_tau, n_outer_cells, n_outer_cells, n_pf_el))
```

Comment:
- if we use frombuffer, do we still need to initialize with zeros above?
- convert to return lists